src/main/java/edu/greenblitz/robotName/commands/arm/roller/RunRoller.java

```diff
@@ -0,0 +1,26 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.EndEffector.RollerUtils.RollerConstants;
+import edu.greenblitz.robotName.subsystems.arm.Wrist;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class RunRoller extends RollerCommand{
+    @Override
+    public void initialize() {
+        roller.rollIn();
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (!Wrist.getInstance().isObjectInside()){
+            new WaitCommand(0.5);
```

Comment:
what is 0.5? if it is a constant move to constants file. (something like .../IntakeConstants -> TIME_AFTER_OBJECT_IN_ROBOT)